pkg/anonymization/anonymizer.go

```diff
@@ -231,6 +245,39 @@ func (anonymizer *Anonymizer) ObfuscateIP(ipStr string) string {
 	return "::"
 }
 
+func (anonymizer *Anonymizer) StoreTranslationTable() *corev1.Secret {
+	if len(anonymizer.translationTable) == 0 {
+		return nil
+	}
+	defer anonymizer.ResetTranslationTable()
```

Comment:
Why do we need to reset it btw?

Reply:
so that only the latest archive's IPs are present in it. Maybe its not clear for me how the anonymizer uses the translation tables, as I understand it we never prune the translation table, we just check if an IP is already in it so theoretically the translation table constantly grows even if some IPs in it are not used anymore.